### ISSUE TITLE
Incorrect block placement warning with inline lambda call

### ISSUE THREAD

**Issue by krzys-h** [colobot/colobot-lint#16]
See colobot/colobot#1150

```C++
auto sth = []()
{
    return 5;
}();
```

**Comment by krzys-h** [comment]
Whoops, I was looking at a wrong error all this time.
